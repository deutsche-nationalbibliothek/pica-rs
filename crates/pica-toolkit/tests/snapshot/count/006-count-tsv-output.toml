bin.name = "pica"
args = "count --tsv -o count.tsv"
status = "success"
stderr = ""
stdout = ""
