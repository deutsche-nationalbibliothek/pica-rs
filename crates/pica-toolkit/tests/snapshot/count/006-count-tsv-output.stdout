records	fields	subfields
1	28	56
